components/brave_wallet_ui/page/screens/onboarding/restore-from-recovery-phrase/restore-from-recovery-phrase.tsx

```diff
@@ -68,7 +68,7 @@ const RESTORE_FROM_EXTENSION_STEPS: RestoreFromOtherWalletSteps[] = [
 
 interface Props {
   restoreFrom:
-  | 'metamask'
+               | 'metamask'
```

Comment:
Any idea what caused the indentation here?

Reply:
Looks like it's to be inline with the property name

Reply:
Nope, I tested and it isn't the pipe - however

```
interface Props {
  restoreFrom: 'metamask'
```

doesn't get any weird formatting. I'm going to change these two and mark this as resolved.